.github/workflows/schema-conformance-action.yml

```diff
@@ -44,3 +48,7 @@ jobs:
           saas-url: https://api.dev.levo.ai
           report: false
           cli_extra_args: "-v INFO"
+      - id: list-files
```

Comment:
@mgiamberardino do you want to leave these?

Reply:
No, It's still WIP. I'm finishing the action configuration. It's not in draft since I need the testing actions to run.